src/styles/header.scss

```diff
@@ -0,0 +1,67 @@
+@use './variables.scss' as *;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  width: auto;
+  background-color: #fff;
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+  &__nav {
+    display: block;
```

Comment:
The display property is set to 'block' for the header__nav. This might not be necessary since the default display for a <nav> element is block. Consider removing it unless you have specific styling needs.